extensions/roc-plugin-style-css/src/css/pipeline.js

```diff
@@ -10,12 +12,20 @@ export default function cssPipeline(base, loaders, isDist, cssModulesEnabled = t
             moduleSettings += '[path]_[name]__[local]___[hash:base64:5]';
         }
     }
+    if (sourceMap) {
+        sourceMapSettings = 'sourceMap';
```

Comment:
What do you think about making this a ternary, a const and conditionally add the `&` to avoid getting a `?&`.

Something like this:
```js
const sourceMapSettings = sourceMapEnabled ?
    'sourceMap&' :
    '';
```